example/go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/containerd/continuity v0.2.2 // indirect
 	github.com/containerd/ttrpc v1.1.0 // indirect
 	github.com/containerd/typeurl v1.0.2 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.1 // indirect
```

Comment:
Ah let me bump to latest stable 2.0.3

Reply:
Actually this is an indirect dependency, should we bump cobra @thaJeztah ?

Reply:
Hm.. good one. Honestly, I think all our CLI's are already on "current" versions of Cobra, so perhaps it's "fine".

OTOH, we don't _need_ "latest" Cobra for this tool, but we _DO_ (likely) want to have the latest `go-md2man` because we know it has fixes in man-page generating.

So I would be fine starting with updating the dependency here (even if it's indirect); for the "paranoid", you could add a comment, e.g. (`// indirect // updated to latest, which has some bug fixes for man-pages`)

Perhaps update to v1.6.0 or v1.7.0? v1.6.0 removes the old YAML v2 dependency, and we should update mousetrap to v1.1.0 to remove some legacy code.

Reply:
Looks like the oldest branch still maintained in docker/cli (23.0) is at v1.6.1, so that could work without disrupting anything; https://github.com/docker/cli/blob/23.0/vendor.mod#L35